api/src/geolocate/get.handler.ts

```diff
@@ -5,19 +5,24 @@ import { ExecuteStatementRequest, SqlParametersList } from 'aws-sdk/clients/rdsd
 import { CORS_HEADERS } from '../util';
 
 const SCHEMA = 'public';
+const ID_FIELD = 'id';
 
 /**
  * Returns all geo identifiers that intersect with a lat, long.
  * @param rdsService: RDS service to connect to the db.
  * @param params: with lat and long coordinates sued to query tables.
  * @param geoid: identifier to retrieve for a table.
  * @param table: db table to query.
+ * @param orderByField: field to sort results by.
+ * @param orderByAsc: whether to sort results in ascending order.
  */
 async function getGeoDataForLatLong(
   rdsService: RDSDataService,
   params: SqlParametersList,
   geoid: string,
   table: string,
+  orderByField: string = ID_FIELD,
+  orderByAsc: boolean = true,
```

Comment:
Does this ever resolve to true?

Reply:
We want this true by default I think - just orders IDs by ascending order. I don't think it actually matters if we are just ordering by ID but this makes it so that the non water system selectors aren't changed at all.